test/bare.js

```diff
@@ -28,7 +28,7 @@ test('bare', function (t) {
             }
         });
         vm.runInNewContext(body, {
-            Buffer: Buffer,
+            Buffer: function (s) { return Buffer.from(s) },
```

Comment:
`Buffer.from` does not exist in old Node versions which browserify still supports—for this change, we need to check if it exists:
```js
var buf = Buffer.from ? Buffer.from(s) : Buffer(s)
```

or use a module like [`safe-buffer`](https://github.com/feross/safe-buffer) that polyfills the `from` method.

Reply:
Fixing. I have some fresh commits pending the PR so this is all backwards compatible.

Reply:
Would you prefer I use safe-buffer to what I patched up in three of the test files (bare.js, bare_shebang.js and buffer.js)? I can do that too.

Reply:
safe-buffer might be easier because you can just use Buffer.from in all the callsites and it'll do the right thing. I don't really mind either way though!